.github/workflows/mtags-auto-release.yml

```diff
@@ -0,0 +1,31 @@
+name: "Mtags auto release"
+on:
+  workflow_dispatch:
+    inputs:
+      scala_version:
+        description: "Scala Version"
+        required: true
+      metals_version:
+        description: "Metals Version"
+        required: true
+        default: "v0.10.9"
```

Comment:
😅 make sure to add bumping this to the next version during an actual full release to the checklist